tests/integration/lambdas/lambda_function.py

```diff
@@ -0,0 +1,7 @@
+import json
```

Comment:
Can we rename this file to `lambda_echo.py`, to be more descriptive and in line with the other files in this directory? (all files in this folder are lambda functions, after all)